tests/README.md

```diff
@@ -112,8 +151,34 @@ XIt("runs some test", func(){
 })
 ```
 
-## Step 4 - Delete kind cluster
+The same can be achieved by using Ginkgo flags. To run a specific test using flags, use the GINKGO_FLAGS variable, e.g.:
 
 ```makefile
-make delete-kind-cluster
+make test TAG=$(whoami) GINKGO_FLAGS='-ginkgo.focus "writes the system info to a results file"'
 ```
+
+or to run all tests with the label "performance":
+
+```makefile
+make test TAG=$(whoami) GINKGO_FLAGS='-ginkgo.label-filter "performance"'
```

Comment:
What do you think about adding a makefile command for this? Given the different test requirements, I think we will need to run each test separately, and this might get a little annoying to copy/paste or type out.

Reply:
I added target for GINKGO_LABEL as well, which can be set to just the label name, e.g. `make test TAG=$(whoami) GINKGO_LABEL=performance`